internal/db/migrations/005_reservations.sql

```diff
@@ -16,6 +16,10 @@ CREATE TABLE aws_reservation_details
   reservation_id BIGINT NOT NULL,
   provider INTEGER NOT NULL DEFAULT provider_type_aws(),
   pubkey_id BIGINT REFERENCES pubkeys(id),
+  source_id INTEGER NOT NULL,
+  instance_type TEXT NOT NULL,
+  amount INTEGER NOT NULL,
+
```

Comment:
Not sure what attributes we should add to those columns
Does all that info required to be saved in the reservation model?
If so, is it because we want a reservation to be saved and has all the info required to start a job?

Reply:
I don't know what attributes are we talking about, but I believe the datatypes seems correct and I don't think it needs more :)
And yes, that is exactly it, we are passing these to the goroutine that executes the actual job and also it might be useful if it fails, because we can easily re-run if required, so it is good to keep the data about the lunch in the DB for a while

Reply:
I mean does "NOT NULL" sufficient here or there are more to be added in your opinion?

Reply:
NOT NULL sounds good enough :)